Documentation/gettingstarted/install-cli.rst

```diff
@@ -8,6 +8,8 @@ and enable/disable a variety of functionality.
     .. code-block:: shell-session
 
       curl -LO https://github.com/cilium/cilium-cli/releases/latest/download/cilium-linux-amd64.tar.gz
+      curl -LO https://github.com/cilium/cilium-cli/releases/latest/download/cilium-linux-amd64.tar.gz.sha256sum
```

Comment:
How about we merge it with the line above?
```
curl -L --remote-name-all https://github.com/cilium/cilium-cli/releases/latest/download/cilium-linux-amd64.tar.gz{,.sha256sum}
```

Reply:
Please also update the `rm` commands at the end of the blocks to remove the SHA-sum file.

Reply:
Thanks, fixed in latest push.